### PR TITLE
improving errors return

### PR DESCRIPTION
**Pull request by manrodrigues**
When a test scenario fails, the assertion message need to be more helpful to identify the problem.

### CODE REVIEW

File: python-test/features/steps/control_plane_agents.py [hunk 1]
```diff
@@ -231,7 +239,7 @@ def reset_agent_remotely(context):
 
 
 @threading_wait_until
-def expect_container_status(token, agent_id, status, event=None):
+def expect_agent_status(token, agent_id, status, event=None):
```
Comment (lpegoraro) [inline python-test/features/steps/control_plane_agents.py:242]: I would add `wait` to the name of the function to distinguish in the name that is a longer duration func.

File: python-test/features/steps/local_agent.py [hunk 2]
```diff
@@ -84,6 +89,7 @@ def run_container_using_ui_command(context, status_port):
     rename_container(context.container_id, LOCAL_AGENT_CONTAINER_NAME + random_string(5))
     if context.container_id not in context.containers_id.keys():
         context.containers_id[context.container_id] = str(context.port)
+    threading.Event().wait(10)
```
Comment (lpegoraro) [inline python-test/features/steps/local_agent.py:92]: Isnt here the `10` to be the seconds from parameter?

**Review by lpegoraro — APPROVED** [review]
Just two comments, but everything else seems good.

File: python-test/features/steps/local_agent.py (hunk 2 above)
Comment (manrodrigues) [reply python-test/features/steps/local_agent.py]: ok.

File: python-test/features/steps/control_plane_agents.py (hunk 1 above)
Comment (manrodrigues) [reply python-test/features/steps/control_plane_agents.py]: check again please

**Review by leoparente — APPROVED** [review]

**Pull request merged**